test/javascripts/acceptance/composer-actions-test.js.es6

```diff
@@ -324,3 +324,30 @@ QUnit.test(
     );
   }
 );
+
+QUnit.test(
+  "replying to post should not trigger an edit conflict",
+  async assert => {
+    // prettier-ignore
+    server.post("/draft.json", request => { // eslint-disable-line no-undef
+      assert.equal(request.requestBody.indexOf("originalText"), -1);
```

Comment:
There's something odd with this `assert`. 

The point of using `async/await` is to write asynchronous code like you would write synchronous code (ie. without callbacks).

Is there a way you could change this so that `assert` is at the end of the function?

Reply:
The problem is that I have to intercept the request and make sure it does include or does not include `originalText` which I do not see how it would be possible otherwise.